tests/samplers_tests/test_samplers.py

```diff
@@ -355,3 +356,24 @@ def objective(trial: Trial, base_value: float) -> float:
     # Non NaN objective value.
     study.optimize(lambda t: objective(t, 1), n_trials=1, catch=())
     assert int(study.best_value) == 1
+
+
+@parametrize_sampler
+def test_partial_fixed_sampling(sampler_class: Callable[[], BaseSampler]) -> None:
+
+    study = optuna.create_study(sampler=sampler_class())
+
+    def objective(trial: Trial) -> Any:
```

Comment:
The Retuned type is supposed to be `float` instead of `Any`?

Reply:
Thank you!